packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -183,7 +183,7 @@ function FolderImpl({ item, anchors }: FolderProps): ReactElement {
           )}
         />
       </ComponentToUse>
-      <Collapse className="ltr:_pr-0 rtl:_pl-0 _pt-1" isOpen={open}>
+      <Collapse className="_p-1" isOpen={open}>
```

Comment:
To resolve a situation like this 

![image](https://github.com/user-attachments/assets/f2da810a-dba0-4e26-9255-b2af47e5d44a)